src/Response/CSVResponse.php

```diff
@@ -0,0 +1,121 @@
+<?php declare(strict_types = 1);
```

Comment:
What are the conventions in opensource php community? In Gamee, Monday Factory, we use:

```php
<?php

declare(strict_types=1);
```
So does for example `consistence/consistence`.

Reply:
It's based on slevomat/coding-standards. :(

Reply:
Right, ok.

Reply:
> It's based on slevomat/coding-standards. :(

Just for reference, this statement is incorrect - Slevomat CS is only a set of rules and this sniff is configurable. Doctrine also uses Slevomat CS with no spaces around `=`:
https://github.com/doctrine/coding-standard/blob/0469c18a1a4724c278f2879c0dd7b1fa860b52de/lib/Doctrine/ruleset.xml#L192-208